Documentation/community/governance/commit_access.rst

```diff
@@ -28,7 +28,7 @@ Cilium Committer Grant/Revocation Policy
 ========================================
 
 A Cilium committer is a participant in the project with the ability to
-commit code directly to the master repository. Commit access grants a
```

Comment:
Unfortunately the branch is actually called "master". It would be great if we could rename the branch as well, although I suspect it's not quite as simple as changing the name (there might be scripts that do special case things for the branch name "master") 😢

Reply:
There's only one "repository' but multiple "branches". We will rename the branch soon so I would suggest to leave this word and let it be renamed once we rename the branch as well.

Reply:
being tracked in https://github.com/cilium/cilium/issues/23110 so I will leave it at master for now